Dockerfile.template

```diff
@@ -40,11 +40,7 @@ ENV PHP_INI_UPLOAD_MAX_FILESIZE 2M
 ENV PHP_INI_POST_MAX_SIZE 8M
 ENV PHP_INI_ALLOW_URL_FOPEN 0
 
-RUN sed -i \
-  -e 's/^\(ServerSignature On\)$/#\1/g' \
-  -e 's/^#\(ServerSignature Off\)$/\1/g' \
-  -e 's/^\(ServerTokens\) OS$/\1 Prod/g' \
-  /etc/apache2/conf-available/security.conf
+%EXTRAS%
```

Comment:
you'll probably have to create a second template for the FPM version of your image instead of editing this one ...

Reply:
I want to minimize the number of files to update and avoid any inconsistency between variants